snuba/datasets/storage.py

```diff
@@ -180,7 +180,7 @@ def __init__(
         )
         assert isinstance(schema, WritableTableSchema)
         self.__table_writer = TableWriter(
-            cluster=get_cluster(storage_set_key),
```

Comment:
This is where we were loading a cluster (environment specific config) while initializing the data model, which should be independent from the environment. I think we should avoid this practice and keep the two decoupled as much as possible during the initialization.